### PR TITLE
fix: fix blocks width in mentors section

### LINKED ISSUE #175: Fix mentors section layout

### PR DESCRIPTION
**Pull request by Tedzury**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Related Tickets & Documents

- Related Issue #175 
- Closes #175

## Screenshots, Recordings

Screenshot 2024-05-02 094941

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

**Review by dzmitry-varabei — APPROVED** [review]

**Review by ansivgit — APPROVED** [review]

**Review by Quiddlee — APPROVED** [review]

File: src/features/courses/mentors/mentors.scss
Comment (SpaNb4) [inline src/features/courses/mentors/mentors.scss:1]: Should we do something about the image size?
before:
image


after:
image

Comment (SpaNb4) [reply src/features/courses/mentors/mentors.scss:1]: I also find it weird that on small screens the image moves to the top
image

File: src/features/courses/mentors/mentors.scss [hunk 1]
```diff
@@ -28,12 +28,17 @@
           order: 1;
         }
 
+        @include media-tablet-large {
+          width: 100%;
+        }
+
+
```
[the remark this answers is not in the document]
Comment (Quiddlee) [reply src/features/courses/mentors/mentors.scss:35]: image

**Comment by Tedzury** [comment]
@SpaNb4 Now picture stays at the top for mobile res. Don't know what is wrong with pic at 1 screenshot, I have normal pic size at that width 🤷‍♂️: 
Screenshot 2024-05-03 173045

**Review by ramzes-84 — APPROVED** [review]

**Review by SpaNb4 — APPROVED** [review]

**Pull request merged**